app/concerns/orderable.rb

```diff
@@ -36,6 +36,7 @@ def reorder_others(is_after)
       others.each_with_index do |other, index|
         next if other.order == index
```

Comment:
Should this be:

```ruby
        correct_order = index
        correct_order += 1 if other.is_a? Reply
        next if other.order == correct_order
        other.update!(order: correct_order)
```
? Otherwise I think this `next if` will skip the update incorrectly.

Reply:
(Do we have a test for this method?)

Reply:
Yeah, I think this comment still stands - seems like we're not calculating the index correctly to know we're updating things right here(?)